org.ferdium.Ferdium.yaml

```diff
@@ -94,6 +96,6 @@ modules:
         dest-filename: ferdium.sh
         commands:
           - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
-          - exec zypak-wrapper /app/Ferdium/ferdium "$@"
+          - exec zypak-wrapper /app/Ferdium/ferdium --ozone-platform-hint=$ELECTRON_OZONE_PLATFORM_HINT "$@"
```

Comment:
@lhw, the `--ozone-platform-hint` option is not required anymore in ferdium 6.7.1 and should be removed from the command.

The purpose of the environment variable `ELECTRON_OZONE_PLATFORM_HINT` is, that it is possible to change the ozone platform without having to use any commandline options.

Reply:
Will do so